module/Motion/CameraAction.cpp

```diff
@@ -30,12 +30,24 @@ void CameraAction::run()
     preAR.run();
   }
 
+  // 撮影対象がB（targetがfalse）の場合は、バックでフィグへ接近
+  if(target == false) {
```

Comment:
前のプルリク見逃してたけど、isAとかの方が直感的だと思う。Bにしてtrueの時にしてもいいけど。
```suggestion
  if(isA == false) {
```